docs/guides/docs/guides.json

```
@@ -1,18 +1,22 @@
-{
-  "installation": {
-    "title": "Toolchain Installation",
-    "description": "Install the Fuel toolchain and binaries."
-  },
-  "quickstart": {
-    "title": "Developer Quickstart",
-    "description": "Get started by building a counter dApp in Fuel."
-  },
-  "running_a_node": {
-    "title": "Running a Node",
-    "description": "Run a local Fuel node."
-  },
-  "testnet_migration": {
-    "title": "Testnet Migration",
-    "description": "Sway and SDK's breaking changes reference."
-  }
-}
+{
+  "installation": {
+    "title": "Toolchain Installation",
+    "description": "Install the Fuel toolchain and binaries."
+  },
+  "quickstart": {
+    "title": "Developer Quickstart",
+    "description": "Get started by building a counter dApp in Fuel"
```

```suggestion
    "description": "Get started by building a counter dApp in Fuel."
```
Adding dot at the end of the description.